policyeval/base_worker.go

```diff
@@ -103,59 +103,65 @@ func (w *BaseWorker) handlePolicy(ctx context.Context, eval *sdk.ScalingEvaluati
 	}
 
 	logger := w.logger.With("policy_id", eval.Policy.ID, "target", eval.Policy.Target.Name)
-	checks := make(map[string]*checkHandler)
-
 	logger.Debug("received policy for evaluation")
 
-	handlersCtx, cancel := context.WithCancel(ctx)
-	defer cancel()
+	// Dispense taget plugin.
+	targetPlugin, err := w.pluginManager.Dispense(eval.Policy.Target.Name, plugins.PluginTypeTarget)
+	if err != nil {
+		return fmt.Errorf(`target plugin "%s" not initialized: %v`, eval.Policy.Target.Name, err)
+	}
+	targetInst := targetPlugin.Plugin().(target.Target)
```

Comment:
i feel like we should perform a type assertion here, even if we know it will be a `target.Target` returned from `Dispense()`. otherwise, we'll get a panic.